### PR TITLE
Updating readme instead of adding conda to path

### PR DESCRIPTION
**Pull request by dparkar**
#127 #128 #97 

@brcrista @DavidStaheli This might help instead.

### CODE REVIEW

File: ubuntu/14.04/standard/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM microsoft/vsts-agent:ubuntu-14.04
 
 # Install basic command-line utilities
 RUN apt-get update \
- && apt-get install -y --no-install-recommends \
+    && apt-get install -y --no-install-recommends \
```
Comment (dparkar) [inline ubuntu/14.04/standard/Dockerfile]: whoops .. reverting

**Comment by brcrista** [comment]
I'll let @dakale make the call on whether to include this in the README.  As far as Conda goes, I feel like the  documentation already makes this clear: https://docs.microsoft.com/en-us/vsts/pipelines/tasks/package/conda-environment?view=vsts

File: ubuntu/14.04/standard/Dockerfile (hunk 1 above)
Comment (dparkar) [reply ubuntu/14.04/standard/Dockerfile]: actually this follows the pep8 standard ...would you rather have these changes or revert ?

Comment (dakale) [reply ubuntu/14.04/standard/Dockerfile]: I would rather revert it. This isnt a python project so we have no reason to follow pep8

**Comment by dakale** [comment]
Can you change this to follow the example of Ruby and Python from the README: 
https://github.com/Microsoft/vsts-agent-docker/blame/master/README.md#L152

```
Python 2.7.15, 3.4.8, 3.5.5, 3.6.5 and 3.7.0 (available through the Use Python Version task)
Ruby 2.3.7, 2.4.4 and 2.5.1 (available through the Use Ruby Version task)
```

Id rather have a link to the Conda task docs linked here. It does more than your change because it describes how to actually use the capability, instead of simply calling out that it might not be on the path

**Comment by dparkar** [comment]
Updated and also left the original note.

**Pull request closed without merging**